circuits/cpp/bootstrap.sh

```diff
@@ -6,7 +6,7 @@ set -eu
 export WASI_VERSION=12
 
 # Update the submodule
-git submodule update --init --recursive
+# git submodule update --init --recursive
```

Comment:
Is this something that will need to be un-commented, once you're done?

Reply:
That's right.